src/components/biggive-form-field-text-input/biggive-text-input.tsx

```diff
@@ -1,16 +1,37 @@
-import { Component, Host, h } from '@stencil/core';
+import { Component, h, Prop } from '@stencil/core';
```

Comment:
This component is not currently used in either donate-frontend or wordpress, so seems to be available for the new design. Although it probably would have made as much sense to start a brand new component with a new name to not even have to check that.